solidity/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "chainlinked",
+  "version": "1.0.4",
```

Comment:
I have concerns about having two `package.json` files and its effect on development.

The top level `/package.json` has everything that's needed for the GUI and solidity development:
```json
  "dependencies": {
    "@ensdomains/ens": "^0.1.1",
    "change-case": "^3.0.2",
    "formik": "^1.0.3",
    "link_token": "^1.0.1",
    "openzeppelin-solidity": "^1.12.0",
    "solidity-cborutils": "github:smartcontractkit/solidity-cborutils"
  },
```

Not to mention the devDependencies. I bring this up because it will dictate what runs when we do `npm run xxx` (or `yarn run xxx`).

I bring this ups because having two will be confusing, especially if the one in `solidity/package.json` does not actually pertain to development in the `solidity/` folder. Counterintuitively `/package.json` is used to run tests for `solidity/`.

Does zeppelinOS require this `package.json`?

Reply:
The only requirement here is that the package has to be published to npm for ZeppelinOS to pick it up.

https://docs.zeppelinos.org/docs/publishing.html

I'm completely open to any ideas for getting the top-level chainlink package published.